client/state/geolocation/schema.js

```diff
@@ -0,0 +1,12 @@
+export const geolocationSchema = {
+	type: 'object',
+	properties: {
+		latitude: { type: 'string' },
+		longitude: { type: 'string' },
+		countryShort: { type: 'string' },
+		countryLong: { type: 'string' },
+		region: { type: 'string' },
+		city: { type: 'string' }
```

Comment:
What we're currently storing doesn't seem particularly useful, so you'll likely want to update that.